tests/data/packages/Local/setup.py

```diff
@@ -0,0 +1,6 @@
+from setuptools import setup
```

Comment:
rather than making a new one, use singlemodule.

Its in an index in data/packages/ versions 0.0.0 and 0.0.1 and it's source is in data/src/singlemodule. So you can constrain to version 0.0.0 and use -f data/packages --no-index -e src/singlemodule.